evaluation/gpqa/run_infer.py

```diff
@@ -40,52 +40,82 @@
 from opendevin.core.logger import get_console_handler
 from opendevin.core.logger import opendevin_logger as logger
 from opendevin.core.main import run_agent_controller
+from opendevin.events.action import Action, AgentFinishAction, MessageAction
+from opendevin.events.observation import Observation
 from opendevin.llm.llm import LLM
 
-AGENT_CLS_TO_FAKE_USER_RESPONSE_FN = {
-    'CodeActAgent': codeact_user_response,
-    'MonologueAgent': monologue_user_response,
-}
+ACTION_FORMAT = """
+<<FINAL_ANSWER||
+<insert correct answer here, must be one of A, B, C, D> (Please dont use any additional characters. Just the letter of the correct answer (A/B/C/D).)
+||FINAL_ANSWER>>
+""".strip()
```

Comment:
@neubig is this similar to what you have in mind for deduplication?

Reply:
AFAIK these parts are also duplicated?

```
        'Again you are being told a million times to first report the answer in the requested format (see again below for reference) before exiting. DO NOT EXIT WITHOUT REPORTING THE ANSWER FIRST.\n'
        'That is, when you have decided on the answer report in the following format:\n'
```

But this is not super-important, please don't let me block you.